Framework/test/testMonitorObject.cxx

```diff
@@ -61,4 +67,48 @@ BOOST_AUTO_TEST_CASE(mo_check)
   BOOST_CHECK_EQUAL(obj.getQuality(), Quality::Null);
 }
 
+BOOST_AUTO_TEST_CASE(mo_save)
+{
+  string objectName = "asdf";
+  TH1F h(objectName.data(), objectName.data(), 100, 0, 99);
+  o2::quality_control::core::MonitorObject obj(&h, "task");
+  cout << "getName : '" << obj.getName() << "'" << endl;
+  cout << "GetName : '" << obj.GetName() << "'" << endl;
+  cout << "title : '" << obj.GetTitle() << "'" << endl;
+  BOOST_CHECK_EQUAL(obj.getName(), "asdf");
+  BOOST_CHECK_EQUAL(obj.GetName(), "asdf");
+  BOOST_CHECK_EQUAL(obj.GetTitle(), "");
+  obj.setIsOwner(false);
+  string libName = "libraryName";
+  obj.addCheck("name", "className", libName);
+  string libName2 = "libraryName2";
+  obj.addCheck("name2", "className2", libName2);
+  obj.setQualityForCheck("name", Quality::Good);
+  BOOST_CHECK_EQUAL(obj.getQuality(), Quality::Good);
+  cout << "quality : " << obj.getQuality() << endl;
+  cout << "check numbers : " << obj.getChecks().size() << endl;
+  CheckDefinition c = obj.getCheck("name2");
+  cout << "check2 libraryName : " << c.libraryName << endl;
+  TFile file("/tmp/test.root", "RECREATE");
+  obj.Write(obj.getName().data());
+  file.Close();
+
+  cout << "***" << endl;
+  TFile file2("/tmp/test.root");
```

Comment:
If there are several builds/test running on the same CI machine, wouldn't that cause some race conditions? Though I don't have an idea how to solve it and anyway they would be very rare probably. Maybe a random number in the name?

Reply:
A random number would do it. I will create a separate PR so that this one can go ahead

Reply:
Thank you , this is a good remark.